sstables/compaction.cc

```diff
@@ -715,7 +715,7 @@ class regular_compaction : public compaction {
                 _schema->full_slice(),
                 service::get_local_compaction_priority(),
                 no_resource_tracking(),
-                nullptr,
+                tracing::trace_state_ptr(),
```

Comment:
These two are equivalent AFAIK.

Reply:
Yes, this is only to improve readability, it more explicitly says "no tracing" (similar to `no_resource_tracking()` which could also be replaced by `{}`).
With `nullptr` I need to go to the definition to see what it is.